eth/handler.go

```diff
@@ -479,6 +479,10 @@ func (h *handler) BroadcastTransactions(txs types.Transactions) {
 	)
 	// Broadcast transactions to a batch of peers not knowing about it
 	for _, tx := range txs {
+		// L1 messages are not broadcast to peers
```

Comment:
```suggestion
		// L1 messages are not broadcasted to peers
```

Reply:
Actually the past tense of broadcast is broadcast (although some people use broadcasted).